test/unit/app/lib/send-email.test.js

```diff
@@ -82,12 +139,34 @@ describe('Send email test', () => {
       }
 
       const result = await sendEmail.sendFarmerEndemicsClaimConfirmationEmail(data, templateId)
+      expect(result).toBe(true)
+      expect(sendSFDEmail).toHaveBeenCalledWith(templateId, data.orgData.orgEmail, { personalisation: expectedPersonalisation, reference: data.reference })
+    })
 
+    test('sendFarmerEndemicsClaimConfirmationEmail sends carbon copy email to organization email', async () => {
```

Comment:
Could you revisit this one? The test title says 'sends carbon copy email to organization email', but it is not asserting that anything is sent to the carbon copy email address.